.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,89 @@
+name: Report a bug
+description: Found something? Thank you for telling us!
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## System Details
+        Please provide the following information to help us diagnose the issue.
+
+  - type: input
+    id: publisher-version
+    attributes:
+      label: Publisher Version
+      description: "Find this in Positron/VS Code: Extensions view → Posit Publisher → under Installation on the right"
+      placeholder: "e.g., 1.18.0"
+    validations:
+      required: true
```

```suggestion
      required: false
```

IMHO we shouldn't _require_ any of these fields. We don't want to put up barriers to people submitting things (even if that information would be very helpful to us)

---

Done in https://github.com/posit-dev/publisher/pull/2826/commits/a29b22254ed81801a8e9ad5a2d0ddad4a2ce9766